metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -107,6 +108,15 @@ object UserConfiguration {
           |separators (even on Windows).
           |""".stripMargin
       ),
+      UserConfigurationOption(
+        "ammonite-jvm-properties",
+        """`[]`.""",
```

Comment:
I noticed with other parameters there is always a dot in the end, any idea why? Just curious

Reply:
Not sure if really needed, but this is rendered here: https://scalameta.org/metals/docs/editors/vim.html#gradle-script

Reply:
🤔 I don't think the dot is needed.